src/gerbil/runtime/error.ss

```diff
@@ -129,8 +129,12 @@ namespace: #f
 ;; method implementations
 (defmethod {:init! Error}
   (lambda (self message . rest)
-    (unchecked-slot-set! self 'message message)
-    (apply class-instance-init! self rest)))
+    (let (message
+          (if (string? message)
+            message
+            (call-with-output-string "" (cut display message <>))))
```

Comment:
this is a sanity check, as we send the message over the wire for actor error results and a bug could end up something unintended; this is **guaranteed** to be a string now.